Bezier Mesh Tool using splines package (#437)
Unity-Technologies/com.unity.probuilder: CHANGELOG.md

### Changes

### Features

> - Added a `New Bezier Mesh` tool that allows users to create a spline(s) and have a `ProbuilderMesh` extruded along the spline(s).

Reviewer: ```suggestion
- Added the `New Bezier Mesh` tool that allows users to create a spline with a `ProbuilderMesh` extruded along the spline.
```
